client/client_interfaces.go

```diff
@@ -115,7 +115,9 @@ type ImageAPIClient interface {
 	ImageCreate(ctx context.Context, parentReference string, options image.CreateOptions) (io.ReadCloser, error)
 	ImageHistory(ctx context.Context, image string, opts image.HistoryOptions) ([]image.HistoryResponseItem, error)
 	ImageImport(ctx context.Context, source image.ImportSource, ref string, options image.ImportOptions) (io.ReadCloser, error)
+	// Deprecated: Use [Client.ImageInspect] instead.
```

Comment:
Perhaps add a hint to use it with the `[client. ImageInspectWithRawResponse]` to read the raw response.

Reply:
Done